custom/icds_reports/tasks.py

```diff
@@ -545,13 +545,17 @@ def _run_custom_sql_script(commands, day=None, db_alias=None):
 @track_time
 def aggregate_awc_daily(day):
 
-    agg_daily_dates = [force_to_date(day) - timedelta(days=2),
-                       force_to_date(day) - timedelta(days=1),
-                       force_to_date(day)]
+    agg_daily_dates = [{
+        'date': force_to_date(day) - timedelta(days=2),
+        'use_agg_awc': False},
+        {'date': force_to_date(day) - timedelta(days=1),
+        'use_agg_awc': False},
+        {'date': force_to_date(day),
+         'use_agg_awc': True}]
 
     for daily_date in agg_daily_dates:
         with transaction.atomic(using=router.db_for_write(AggAwcDaily)):
-            AggAwcDaily.aggregate(daily_date)
+            AggAwcDaily.aggregate(**daily_date)
```

Comment:
nit: i think this would be slightly more clear if you unpacked the dict yourself and passed in the values, rather than using `**`
```
for day in agg_daily_dates:
    with transaction.atomic(using=router.db_for_write(AggAwcDaily)):
        AggAwcDaily.aggregate(date=day['date'], use_agg_awc=day['use_agg_awc'])
```
or even just had the last line be
```
AggAwcDaily.aggregate(day['date'], day['use_agg_awc'])
```
which might be just as clear since the keys have the same name as the kwargs
but that may be a personal preference, not sure if cory would agree.

Reply:
It's the second zen of python! (kind of)

> Explicit is better than implicit.

(but I don't feel strongly)